smoke-test/src/test/java/org/opennms/smoketest/OpenNMSCloudPluginIT.java

```diff
@@ -0,0 +1,161 @@
+/*******************************************************************************
+ * This file is part of OpenNMS(R).
+ *
+ * Copyright (C) 2022 The OpenNMS Group, Inc.
+ * OpenNMS(R) is Copyright (C) 1999-2022 The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is a registered trademark of The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * OpenNMS(R) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with OpenNMS(R).  If not, see:
+ *      http://www.gnu.org/licenses/
+ *
+ * For more information contact:
+ *     OpenNMS(R) Licensing <license@opennms.org>
+ *     http://www.opennms.org/
+ *     http://www.opennms.com/
+ *******************************************************************************/
+
+package org.opennms.smoketest;
+
+import static com.jayway.awaitility.Awaitility.await;
+import static java.lang.String.format;
+import static java.util.concurrent.TimeUnit.MINUTES;
+import static java.util.concurrent.TimeUnit.SECONDS;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.time.Duration;
+
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.opennms.smoketest.stacks.OpenNMSStack;
+import org.opennms.smoketest.stacks.StackModel;
+import org.opennms.smoketest.utils.KarafShellUtils;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.utility.DockerImageName;
+import org.testcontainers.utility.MountableFile;
+
+import com.github.dockerjava.api.command.CreateContainerCmd;
+
+public class OpenNMSCloudPluginIT {
+
+    private static final String PATH_TO_FILE = "target/plugin-cloud-assembly/org.opennms.plugins.cloud-assembly.kar";
+    private static final String PATH_TO_CLOUD_MOCK = "target/cloud-mock-with-dependencies/org.opennms.plugins.cloud-mock-with-dependencies.jar";
+    private static final String CONTAINER_PATH = "/usr/share/opennms/deploy/";
+    private static final String CLOUD_MOCK_JAR_NAME = "org.opennms.plugins.cloud-mock-with-dependencies.jar";
+    private static final String CLOUD_MOCK_MAIN = "org.opennms.plugins.cloud.ittest.MockCloudMain";
+    private static final String KAR_FILE_NAME = "org.opennms.plugins.cloud-assembly.kar";
+
+    @ClassRule
+    public static OpenNMSStack stack = OpenNMSStack
+            .withModel(StackModel.newBuilder()
+                    .withSentinel()
+                    .build());
```

Comment:
There is `OpenNMSStack.SENTINEL` for this.

Reply:
ok